sourcecode/chesssimul/board.py

```diff
@@ -222,8 +235,50 @@ def move_bishop(self, move: Move) -> bool:
     def move_pawn(self, move: Move) -> bool:
         start_square: Square = self.board[move.start[0]][move.start[1]]
         end_square: Square = self.board[move.end[0]][move.end[1]]
-        # if move.end[0] == move.start[0] + 2 or move.end[0] == move.start[0] - 2:
-        return False
+        # Pawns can only move forward one square at a time
+        if start_square.piece.color == Color.BLACK:
```

Comment:
To save some comparison operation, you could check here that move.start[1] == move.end[1] (i.e the squares are in the same column) and assume it as true in the next section